tensorflow/python/autograph/converters/break_statements.py

```diff
@@ -145,7 +154,18 @@ def visit_For(self, node):
       new_for_node = node[1]
       anno.setanno(new_for_node, 'extra_test', extra_test)
```

Comment:
These two annotations need to be copied over (use `anno.copyanno` in both cases)

Reply:
Added.